libs/ui/design-system/src/lib/components/DataGrid/DataGrid.tsx

```diff
@@ -65,6 +66,11 @@ export function DataGrid<T>({
   const [sortColumns, setSortColumns] = useState<SortColumn[]>([]);
   const [rows, setRows] = useState<T[]>(incomingRows);
 
+  const visibleColumns = useMemo(
+    () => columns.filter((c) => !c.hidden),
```

Comment:
If "hidden" is undefined (it is optional), wouldn't `!c.hidden` be truth-y ? Is that what you mean to happen?

Reply:
undefind === not hidden === visible